### ISSUE TITLE
Add iterations validation for phppass

### ISSUE THREAD

**Issue by JustGuardian** [acmesecorg/metacrack#5]
As for the validator of the iterations in bcrypt and PBKDF2-SHA256, meta needs a validator for iterations in hashes like phppass.

**Comment by jameswestgate** [comment]
Do you have the mode / hash numbers you would like to see this added for ? The iterator has to be parsed out of each hash format, so will be done on a case by case basis for now.

**Comment by JustGuardian** [comment]
doing case by case could be the best option I think

**Comment by jameswestgate** [comment]
Which hash modes please :) 
Ah sorry, the title  of this is phppass
You can see the supported modes here https://github.com/acmesecorg/metacrack/blob/rocksdb/PluginBase.cs#L269

**Comment by JustGuardian** [comment]
yeah i know that it is supported, but I meant the validator for different types of iterations in the hash
`$H$`
`$P$`
etc
